Makefile

```diff
@@ -417,7 +417,7 @@ docker-presto-cleanup:
 	make clear-testing
 
 docker-presto-up: docker-metastore-setup docker-presto-setup
-	docker-compose -f ./testing/compose_files/docker-compose-presto.yml up -d
+	docker-compose -f ./testing/compose_files/docker-compose-presto.yml up -d $(build)
```

Comment:
If this is getting build would you mind making a `-no-build` version of this and using that in the other presto `-no-build` options?

Reply:
no-build is default.
To build:
`make docker-presto-up build=--build`